porcupine/plugins/update_check.py

```diff
@@ -0,0 +1,137 @@
+from __future__ import annotations
+
+import datetime
+import logging
+
+import requests
+
+from porcupine import __version__ as this_porcupine_version
+from porcupine import settings, utils
+from porcupine.plugins import statusbar
+from porcupine.settings import global_settings
+
+log = logging.getLogger(__name__)
+
+# Place the update checkbox towards the end of the settings dialog
+setup_after = ["restart"]
+
+
+def x_days_ago(days: int) -> str:
+    days_in_year = 365.25  # good enough
+    days_in_month = days_in_year / 12
+    months = round(days / days_in_month)
+
+    if days == 0:
+        return "today"
+    if days == 1:
+        return "yesterday"
+    if days < 1.5 * days_in_month:
+        return f"{days} days ago"
+
+    months = round(days / days_in_month)
+    if months < 12:
+        return f"about {months} months ago"
+    if months == 12:
+        return "about 1 year ago"
```

Comment:
If we make a distinction here anyway, wouldn’t it be better, to say "about a year ago"?